src/libraries/System.Private.Xml/tests/Readers/NameTable/TestFiles.cs

```diff
@@ -79,7 +79,7 @@ protected static void DeleteTestFile(string strFileName)
         public static void CreateGenericTestFile(string strFileName)
         {
             MemoryStream ms = new MemoryStream();
-            TextWriter tw = new StreamWriter(ms);
+            using var tw = new StreamWriter(ms, encoding:null, bufferSize:-1, leaveOpen:true);
```

Comment:
nit: should we put space after `:`?

Reply:
I actually don't know what our prevailing style is. I will have to check. @sharwell is there any StyleCop or editorconfig linting of this available?